extensions/amp-animation/0.1/amp-animation.js

```diff
@@ -408,12 +410,14 @@ export class AmpAnimation extends AMP.BaseElement {
   /**
    * Creates the runner but animations will not start.
    * @param {?JsonObject=} opt_args
+   * @param {?Object=} opt_positionObserverArgs
    * @return {!Promise}
    * @private
    */
-  createRunnerIfNeeded_(opt_args) {
+  createRunnerIfNeeded_(opt_args, opt_positionObserverArgs) {
```

Comment:
`opt_viewportData` is cleaner IMO. Avoiding position-observer naming and logic throughout the amp-animation stack would be nice.

Reply:
Done.